### PR TITLE
adds the moscow mule (port) and fixes sol dry having no sprite

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr https://github.com/tgstation/tgstation/pull/48466/files
moscow mule is  just alcoholic no special effect this time
made with sol dry, vodka, lime juice, and ice

#### Changelog
// maintainer edit: fixed changelog
:cl:  OnlineGirlfriend
rscadd: Added a new alcoholic drink
bugfix: fixed sol drys lack of sprite 
imageadd: added moscow mule sprite
/:cl:

### CODE REVIEW

**Review by JamieD1 — APPROVED** [review]

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]
Remove the new sol dry sprite, I fixed that already.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1714,6 +1714,17 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "Mojito"
 	glass_desc = "A drink that looks as refreshing as it tastes."
 
+/datum/reagent/consumable/ethanol/moscow_mule
+	name = "Moscow Mule"
+	description = "A chilly drink that reminds you of the Derelict."
+	color = "#EEF1AA"
+	boozepwr = 30
+	quality = DRINK_GOOD
+	taste_description = "refreshing spiciness"
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	taste_description = "COMMUNISM"
```

**Review by alexkar598 — APPROVED** [review]

**Comment by alexkar598** [comment]
Fix the corn flakes

**Comment by JamieD1** [comment]
@boodaliboo

**Pull request closed without merging**